### DISCUSSION TITLE
Couriers who are not assigned to a warehouse.

### DISCUSSION THREAD

**Discussion by Mikec78660** [ldtteam/minecolonies#9199] [Ideas]
I'm using minecolonies-1.19.4-1.0.1430-BETA and have set the colony size to the max, set the number of colonist to 500. My goal is to make the largest colony I could. So far so good. What is really giving me problems however are couriers and the warehouse. I had to add a second warehouse just to get more couriers, not because of space. at 350 colonist space wasn't an issue but it was taking 3 or 4 minecraft days to get a delivery. But adding the second warehouse came with it's own issues. It seems if the warehouse closest to the requester didn't have something it wasn't going to the second warehouse but instead to the player. And I couldn't figure out how to balance the couriers at all. I tried different combinations of overlapping couriers and warehouses but I would still end up with one courier having 15 requests and another with zero. 

So my question is would it be possible to have couriers that are not associated with a warehouse who would only do deliveries say from the blacksmith to the guard tower, or from the glassblower to the alchemist? I'm trying to get everything as automated as I can so I can focus on getting colonist as close to their workplace as possible and dealing with raids. But to get a single pain of stained glass to a builder takes a delivery of cobble stone to the crusher to make gravel, then from gavel to sand, then a delivery to the glass blower to make a glass pain, then to the dyer to stain it, then finally to the builder. It would be great to have enough couriers to do that without being limited to the 10 you can have from a single warehouse.

Or if you could allow a courier to pick up or drop off at a warehouse they were not assigned to but I guess that kind of defeats the purpose. I have even tried to put a postbox with a hopper that would dump into the inventory of a courier hut associated with the other warehouse but those items will sit for days before the hut's courier doing anything with them. There is the endchest that is the two warehouses but that still requires me to manually move things around.

I really love minecolonies, just with I could understand the deliveries. I am up to 400 coloniest but it is really at a standstill now as I can pretty much only get food delivered from the ag huts to the restaurants to keep everyone alive. I am having to replace Armour and weapons myself. My blacksmith can build them fast enough but I can't get them delivered. Nothing like seeing a knight dressed like a knight squire with his wooden sword surrounded by 10 barbarians :-)

**Comment by chriskeena** [comment]
A nice adjunct to this would be if unassigned couriers were used to transfer items between warehouses.
